apis/clusters/v1beta1/cassandra_types.go

```diff
@@ -242,6 +273,16 @@ func (cs *CassandraSpec) DebeziumFromInstAPI(iDebeziums []*models.Debezium) (dcs
 	return debeziums
 }
 
+func (cs *CassandraSpec) ShotoverProxyromInstAPI(iShotoverProxys []*models.ShotoverProxy) (sps []ShotoverProxySpec) {
+	var shotoverProxys []ShotoverProxySpec
+	for _, iShotoverProxy := range iShotoverProxys {
+		shotoverProxys = append(shotoverProxys, ShotoverProxySpec{
+			NodeSize: iShotoverProxy.NodeSize,
+		})
+	}
+	return shotoverProxys
+}
+
```

Comment:
you may use here predeclared varialbe `sps` instead of definning `shotoverProxys`

Reply:
fixed